src/ngraph/opsets/opset1.hpp

```diff
@@ -16,7 +16,7 @@
 
 #pragma once
 
-#include "ngraph/ngraph.hpp"
+#include "ngraph/ops.hpp"
```

Comment:
Is this line really needed here?
Can we move this line into `ngraph/opsets/opset1_tbl.hpp`?

Reply:
I think this PR [#3985](https://github.com/NervanaSystems/ngraph/pull/3985) fixed it.